### DISCUSSION TITLE
Meta Data

### DISCUSSION THREAD

**Discussion by jarrodCoombes** [advplyr/audiobookshelf#154] [General]
This might be tangential to this project, but I am wondering how get the book list out of Audiobookshelf and into Calibre so I can get it to generate the needed meta data files (opf file IIRC).

I have many e-books, and the one piece missing from the puzzle is the lack of accurate metadata (most notably the synopsis).

**Comment by Jdiesel87** [comment]
This isn't exactly what you are looking for but you might want to check out this tool originally designed for Booksonic. It allows you to edit the cover art, narrator, and synopsis manually which are output as cover.jpg, reader.txt, and desc.txt that can be used in Audiobookshelf. The tool doesn't scrape the information so you still have to paste it in manually but it is setup better for bulk editing which might make things go a bit faster.

https://github.com/galacticat/booksonic-library-editor 

It also has the option for running scripts. If you are handy you might be able to create a script to covert the opd into those txt files that can be used by Audiobookshelf. Here is an example script for ODM files.

https://github.com/popeen/Booksonic-Export-Booksonic-Metadata-from-ODM-Files/blob/master/odmToBooksonicMeta.ps1

**Comment by advplyr** [comment]
I'm not entirely sure what you mean, but with the synopsis the experimental match feature could be helpful #157 
If this goes well then a next step would be to add a scanner option to allow pulling this info on scans.

**Reply by jarrodCoombes** [reply]
I was mostly asking for a way to get data out of Audiobookshelf and into Calibre to try and avoid having to get the meta data manually for each book. But I think #157 supersedes this.
